app/config/config_prod.yml

```diff
@@ -2,12 +2,6 @@ imports:
     - { resource: config.yml }
     - { resource: security.yml }
 
-framework:
-   validation:
-       cache: validator.mapping.cache.doctrine.apc
```

Comment:
was this intentionally dropped? http://symfony.com/doc/current/reference/configuration/framework.html#reference-validation-cache